macros/optimize_tables.sql

```diff
@@ -99,6 +99,14 @@ OPTIMIZE archipelago_ethereum.events;
 OPTIMIZE foundation_ethereum.events;
 {% endset %}
 
+{% set ironbank_ethereum_itokens %}
```

Comment:
this file isn't necessary as both of your objects are being built as views. this only applies to tables.

Reply:
Fixed.